frontend-web/webclient/app/Applications/Overview.tsx

```diff
@@ -103,7 +114,7 @@ function Applications(props: ApplicationsProps): JSX.Element {
             />
 
             {featuredTags.map(tag =>
-                <TagGrid key={tag} favorites={favPairs} tag={tag} omit={defaultTools} rows={1} columns={7}
```

Comment:
If the backend excludes them, then the `omit` field shouldn't be necessary, should it?